From 191ab1ad011bec64b6374823297f77ac9bcbddd4 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Mon, 16 May 2016 14:01:07 -0700
Subject: [PATCH 0601/1164] Update to 5.9.0-SNAPSHOT on Mon May 16 14:00:09
 PDT 2016

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '333'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hive.git (push)
origin	git@github.sf.cloudera.com:CDH/hive.git (push)
---
 accumulo-handler/pom.xml              |    2 +-
 ant/pom.xml                           |    2 +-
 beeline/pom.xml                       |    2 +-
 cli/pom.xml                           |    2 +-
 common/pom.xml                        |    2 +-
 contrib/pom.xml                       |    2 +-
 hbase-handler/pom.xml                 |    2 +-
 hcatalog/core/pom.xml                 |    2 +-
 hcatalog/hcatalog-pig-adapter/pom.xml |    2 +-
 hcatalog/pom.xml                      |    2 +-
 hcatalog/server-extensions/pom.xml    |    2 +-
 hcatalog/streaming/pom.xml            |    2 +-
 hcatalog/webhcat/java-client/pom.xml  |    2 +-
 hcatalog/webhcat/svr/pom.xml          |    2 +-
 hwi/pom.xml                           |    2 +-
 itests/custom-serde/pom.xml           |    2 +-
 itests/hcatalog-unit/pom.xml          |    2 +-
 itests/hive-jmh/pom.xml               |    2 +-
 itests/hive-minikdc/pom.xml           |    2 +-
 itests/hive-unit-hadoop2/pom.xml      |    2 +-
 itests/hive-unit/pom.xml              |    2 +-
 itests/pom.xml                        |    2 +-
 itests/qtest-spark/pom.xml            |    2 +-
 itests/qtest/pom.xml                  |    2 +-
 itests/test-serde/pom.xml             |    2 +-
 itests/util/pom.xml                   |    2 +-
 jdbc/pom.xml                          |    2 +-
 metastore/pom.xml                     |    2 +-
 odbc/pom.xml                          |    2 +-
 packaging/pom.xml                     |    2 +-
 pom.xml                               |    6 +++---
 ql/pom.xml                            |    2 +-
 serde/pom.xml                         |    2 +-
 service/pom.xml                       |    2 +-
 shims/0.20S/pom.xml                   |    2 +-
 shims/0.23/pom.xml                    |    2 +-
 shims/aggregator/pom.xml              |    2 +-
 shims/common/pom.xml                  |    2 +-
 shims/pom.xml                         |    2 +-
 shims/scheduler/pom.xml               |    2 +-
 spark-client/pom.xml                  |    2 +-
 testutils/pom.xml                     |    2 +-
 42 files changed, 44 insertions(+), 44 deletions(-)

diff --git a/accumulo-handler/pom.xml b/accumulo-handler/pom.xml
index 448f045..4f9a809 100644
--- a/accumulo-handler/pom.xml
+++ b/accumulo-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/ant/pom.xml b/ant/pom.xml
index 72bc161..59c2c95 100644
--- a/ant/pom.xml
+++ b/ant/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/beeline/pom.xml b/beeline/pom.xml
index 8ee2e66..d1c20d1 100644
--- a/beeline/pom.xml
+++ b/beeline/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/cli/pom.xml b/cli/pom.xml
index 2eab1eb..8c34203 100644
--- a/cli/pom.xml
+++ b/cli/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/common/pom.xml b/common/pom.xml
index c27f113..7b4b275 100644
--- a/common/pom.xml
+++ b/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/contrib/pom.xml b/contrib/pom.xml
index f17f852..04b184d 100644
--- a/contrib/pom.xml
+++ b/contrib/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hbase-handler/pom.xml b/hbase-handler/pom.xml
index 862cc69..a212191 100644
--- a/hbase-handler/pom.xml
+++ b/hbase-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/core/pom.xml b/hcatalog/core/pom.xml
index 0461163..455c2dc 100644
--- a/hcatalog/core/pom.xml
+++ b/hcatalog/core/pom.xml
@@ -26,7 +26,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/hcatalog-pig-adapter/pom.xml b/hcatalog/hcatalog-pig-adapter/pom.xml
index dd39686..65c7cbd 100644
--- a/hcatalog/hcatalog-pig-adapter/pom.xml
+++ b/hcatalog/hcatalog-pig-adapter/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/pom.xml b/hcatalog/pom.xml
index 1b2092c..7ef45f7 100644
--- a/hcatalog/pom.xml
+++ b/hcatalog/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/server-extensions/pom.xml b/hcatalog/server-extensions/pom.xml
index a28f361..cfa35ba 100644
--- a/hcatalog/server-extensions/pom.xml
+++ b/hcatalog/server-extensions/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/streaming/pom.xml b/hcatalog/streaming/pom.xml
index 6b1d68e..d7a2fbd 100644
--- a/hcatalog/streaming/pom.xml
+++ b/hcatalog/streaming/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/java-client/pom.xml b/hcatalog/webhcat/java-client/pom.xml
index 77711fc..689d7d6 100644
--- a/hcatalog/webhcat/java-client/pom.xml
+++ b/hcatalog/webhcat/java-client/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/svr/pom.xml b/hcatalog/webhcat/svr/pom.xml
index e817dd1..cfbdd67 100644
--- a/hcatalog/webhcat/svr/pom.xml
+++ b/hcatalog/webhcat/svr/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hwi/pom.xml b/hwi/pom.xml
index e855587..fa22396 100644
--- a/hwi/pom.xml
+++ b/hwi/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-serde/pom.xml b/itests/custom-serde/pom.xml
index fb29a4c..678d393 100644
--- a/itests/custom-serde/pom.xml
+++ b/itests/custom-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hcatalog-unit/pom.xml b/itests/hcatalog-unit/pom.xml
index 71063ab..7b93586 100644
--- a/itests/hcatalog-unit/pom.xml
+++ b/itests/hcatalog-unit/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-jmh/pom.xml b/itests/hive-jmh/pom.xml
index 549dcc0..89c27c8 100644
--- a/itests/hive-jmh/pom.xml
+++ b/itests/hive-jmh/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-minikdc/pom.xml b/itests/hive-minikdc/pom.xml
index 836bcb8..7a4072e 100644
--- a/itests/hive-minikdc/pom.xml
+++ b/itests/hive-minikdc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit-hadoop2/pom.xml b/itests/hive-unit-hadoop2/pom.xml
index d545f28..6da1388 100644
--- a/itests/hive-unit-hadoop2/pom.xml
+++ b/itests/hive-unit-hadoop2/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit/pom.xml b/itests/hive-unit/pom.xml
index d3e53a7..891b57d 100644
--- a/itests/hive-unit/pom.xml
+++ b/itests/hive-unit/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/pom.xml b/itests/pom.xml
index d9a36fd..68291db 100644
--- a/itests/pom.xml
+++ b/itests/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest-spark/pom.xml b/itests/qtest-spark/pom.xml
index 233321a..d8a3159 100644
--- a/itests/qtest-spark/pom.xml
+++ b/itests/qtest-spark/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest/pom.xml b/itests/qtest/pom.xml
index 2ce459f..1277ee1 100644
--- a/itests/qtest/pom.xml
+++ b/itests/qtest/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/test-serde/pom.xml b/itests/test-serde/pom.xml
index db82ece..3319d70 100644
--- a/itests/test-serde/pom.xml
+++ b/itests/test-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/util/pom.xml b/itests/util/pom.xml
index c6e9aa9..775c03f 100644
--- a/itests/util/pom.xml
+++ b/itests/util/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/jdbc/pom.xml b/jdbc/pom.xml
index 98ec378..ee08f3c 100644
--- a/jdbc/pom.xml
+++ b/jdbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/metastore/pom.xml b/metastore/pom.xml
index 135b41a..0d22621 100644
--- a/metastore/pom.xml
+++ b/metastore/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/odbc/pom.xml b/odbc/pom.xml
index b36118d..56199cb 100644
--- a/odbc/pom.xml
+++ b/odbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/packaging/pom.xml b/packaging/pom.xml
index 208e192..fca5a63 100644
--- a/packaging/pom.xml
+++ b/packaging/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/pom.xml b/pom.xml
index 73cd873..f65bfd9 100644
--- a/pom.xml
+++ b/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.8.0-SNAPSHOT</version>
+    <version>5.9.0-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
   <artifactId>hive</artifactId>
-  <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+  <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Hive</name>
@@ -54,7 +54,7 @@
   </modules>
 
   <properties>
-    <cdh.hive.version>1.1.0-cdh5.8.0-SNAPSHOT</cdh.hive.version>
+    <cdh.hive.version>1.1.0-cdh5.9.0-SNAPSHOT</cdh.hive.version>
     <hive.version.shortname>1.1.0</hive.version.shortname>
 
     <!-- Build Properties -->
diff --git a/ql/pom.xml b/ql/pom.xml
index de11873..d4c6786 100644
--- a/ql/pom.xml
+++ b/ql/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/serde/pom.xml b/serde/pom.xml
index 457d5e9..c636ec2 100644
--- a/serde/pom.xml
+++ b/serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/service/pom.xml b/service/pom.xml
index e4a793e..f42f872 100644
--- a/service/pom.xml
+++ b/service/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.20S/pom.xml b/shims/0.20S/pom.xml
index 86d7e11..7640bd5 100644
--- a/shims/0.20S/pom.xml
+++ b/shims/0.20S/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.23/pom.xml b/shims/0.23/pom.xml
index c364354..57f8489 100644
--- a/shims/0.23/pom.xml
+++ b/shims/0.23/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/aggregator/pom.xml b/shims/aggregator/pom.xml
index e7c3c9e..cab0686 100644
--- a/shims/aggregator/pom.xml
+++ b/shims/aggregator/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/common/pom.xml b/shims/common/pom.xml
index 3094909..c2da1e1 100644
--- a/shims/common/pom.xml
+++ b/shims/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/pom.xml b/shims/pom.xml
index c500ab0..0afd815 100644
--- a/shims/pom.xml
+++ b/shims/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/scheduler/pom.xml b/shims/scheduler/pom.xml
index 21f511b..7ae1aa5 100644
--- a/shims/scheduler/pom.xml
+++ b/shims/scheduler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/spark-client/pom.xml b/spark-client/pom.xml
index f00db3e..d430656 100644
--- a/spark-client/pom.xml
+++ b/spark-client/pom.xml
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
diff --git a/testutils/pom.xml b/testutils/pom.xml
index 9b0f0ce..00a82ae 100644
--- a/testutils/pom.xml
+++ b/testutils/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.8.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.9.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-- 
1.7.9.5

